core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneticAlteration.java

```diff
@@ -104,7 +111,7 @@ public int addGeneticAlterations(int geneticProfileId, long entrezGeneId, String
        if (MySQLbulkLoader.isBulkLoad() ) {
           //  write to the temp file maintained by the MySQLbulkLoader
           MySQLbulkLoader.getMySQLbulkLoader("genetic_alteration").insertRecord(Integer.toString( geneticProfileId ),
-                  Long.toString( entrezGeneId ), valueBuffer.toString());
```

Comment:
yes .. entrezGeneId should not be Long anyway ... it has been set to an int(11) mysql field everywhere now
a 4 space indent would be standard, although fixing style is not necessary for this PR. (another example are the extra spaces around argument in the function call)